pkgs/development/python-modules/scikit-build/default.nix

```
@@ -56,6 +59,15 @@ buildPythonPackage rec {
 
   dontUseCmakeConfigure = true;
 
+  # scikit-build PR #458. Remove in version > 0.10.0
+  patches = [
+    (fetchpatch {
+      name = "python38-platform_linux_distribution-fix-458";
+      url = "https://patch-diff.githubusercontent.com/raw/scikit-build/scikit-build/pull/458.patch";
```

the pull url is not stable, as someone could still push to it
```suggestion
      url = "https://github.com/scikit-build/scikit-build/commit/faa7284e5bc4c72bc8744987acdf3297b5d2e7e4.patch";
```